packages/payment/src/models/payment-provider.ts

```diff
@@ -0,0 +1,12 @@
+import { Entity, PrimaryKey, Property } from "@mikro-orm/core"
+
+@Entity({ tableName: "payment_provider" })
+export default class PaymentProvider {
+  @PrimaryKey({ columnType: "text" })
+  id: string
+
+  @Property({
+    default: true,
+  })
+  is_enabled: boolean = true
```

Comment:
todo: should be optional since we have a default value right?

Reply:
in. this case it should be only defined in OptionalProps otherwise if defined as optional with `?` it could be unset?

Reply:
yes optional props only sorry